lib/AbsoluteGrid.jsx

```diff
@@ -60,6 +60,7 @@ export default class AbsoluteGrid extends React.Component {
       var gridItem = React.cloneElement(this.props.displayObject, _.assign(this.props.displayObject.props, {
         style: style,
         item: item,
+        itemsLength: this.props.items.length,
```

Comment:
If we `var len = this.props.items.length;` before the map statement on about line 54, then there's a very minor efficiency enhancement.